src/shared/ui/charts/LineChart/LineChart.tsx

```diff
@@ -47,7 +47,7 @@ const LineChart: FC<Props> = ({ data, config }) => {
     return currentWeekDates.flatMap(currentWeekDate => {
       const currentWeekDayValues = data.map(dataItem =>
         areDatesEqual(dataItem.date, currentWeekDate) &&
-        dataItem.value &&
+        (!!dataItem.value || typeof dataItem.value === 'number') &&
```

Comment:
It is not easy to interpret what this change means. Please refactor this complex condition into single or multiple boolean variables that describe their own meanings